sinks/hawkular/driver.go

```diff
@@ -78,7 +106,11 @@ func (self *hawkularSink) Register(mds []sink_api.MetricDescriptor) error {
 
 // Fetches definitions from the server and checks that they're matching the descriptors
 func (self *hawkularSink) updateDefinitions(mt metrics.MetricType) error {
-	mds, err := self.client.Definitions(metrics.Filters(metrics.TypeFilter(mt)))
+	m := make([]metrics.Modifier, len(self.modifiers), len(self.modifiers)+1)
```

Comment:
`append` doesn't modify the underlying slice. Of course if stuff operates on the elements then that will be reflected in both slices which isn't what you want, but copying for the sake of it is a bit of a waste - but as it's a negligible optimisation I'm not going to argue the point.